src/include/common/container/bitmap.h

```diff
@@ -0,0 +1,113 @@
+#pragma once
+#include <memory>
+#include "common/common_defs.h"
+#include "common/concurrent_bitmap.h"
```

Comment:
Why is this a dependency?

Reply:
BitmapSize(), but in the next comment you're suggesting I reverse it and have bitmap.h contain the common stuff and have ConcurrentBitmap get it from there.